PowerShell/Azure/Storage/Set-AzureWebAppStorageContentFromStorageWithAzCopy/Set-AzureWebAppStorageContentFromStorageWithAzCopy.psm1

```diff
@@ -0,0 +1,220 @@
+
+.EXAMPLE
+    $setStorageContentParameters = @{
+        ResourceGroupName = "CoolStuffHere"
+        WebAppName = "NiceApp"
+        SourceConnectionStringName = "SourceStorage"
+        DestinationConnectionStringName = "DestinationStorage"
+    }
+    Set-AzureWebAppStorageContentFromStorageWithAzCopy @setStorageContentParameters
+
+.EXAMPLE
+    $setStorageContentParameters = @{
+        ResourceGroupName = "CoolStuffHere"
+        WebAppName = "NiceApp"
+        SourceConnectionStringName = "SourceStorage"
+        DestinationConnectionStringName = "DestinationStorage"
+        ContainerWhiteList = @("media", "stuff")
+    }
+    Set-AzureWebAppStorageContentFromStorageWithAzCopy @setStorageContentParameters
+
+.EXAMPLE
+    $setStorageContentParameters = @{
+        ResourceGroupName = "CoolStuffHere"
+        WebAppName = "NiceApp"
+        SourceConnectionStringName = "SourceStorage"
+        DestinationConnectionStringName = "DestinationStorage"
+        ContainerWhiteList = @("media", "stuff")
+        SasLifetimeMinutes = 10
+    }
+    Set-AzureWebAppStorageContentFromStorageWithAzCopy @setStorageContentParameters
+#>
+
+Import-Module Az.Storage
+
+function Set-AzureWebAppStorageContentFromStorageWithAzCopy
+{
+    [CmdletBinding()]
+    [Alias('sascsazc')]
+    Param
+    (
+        [Alias('ResourceGroupName')]
+        [Parameter(
+            Mandatory = $true,
+            HelpMessage = 'You need to provide the name of the Resource Group the Source Web App is in.')]
+        [string] $SourceResourceGroupName,
+
+        [Alias('WebAppName')]
+        [Parameter(Mandatory = $true, HelpMessage = 'You need to provide the name of the Web App.')]
+        [string] $SourceWebAppName,
+
+        [Alias('SlotName')]
+        [Parameter(HelpMessage = 'The name of the Source Web App slot.')]
+        [string] $SourceSlotName,
+
+        [Alias('ConnectionStringName')]
+        [Parameter(
+            Mandatory = $true,
+            HelpMessage = 'You need to provide a connection string name for the source Storage Account.')]
+        [string] $SourceConnectionStringName,
+
+        [Parameter(HelpMessage = 'The name of the Destination Resource Group if it differs from the Source.')]
+        [string] $DestinationResourceGroupName = $SourceResourceGroupName,
+
+        [Parameter(HelpMessage = 'The name of the Destination Web App if it differs from the Source.')]
+        [string] $DestinationWebAppName = $SourceWebAppName,
+
+        [Parameter(HelpMessage = 'The name of the Destination Web App Slot if it differs from the Source.')]
+        [string] $DestinationSlotName = $SourceSlotName,
+
+        [Parameter(HelpMessage = 'The name of the Destination Connection String if it differs from the Source.')]
+        [string] $DestinationConnectionStringName = $SourceConnectionStringName,
+
+        [Parameter(HelpMessage = 'A list of names of Blob Containers to include.')]
+        [string[]] $ContainerWhiteList = @(),
```

Comment:
I guess you mean `$ContainerIncludeList`, but yeah. This cmdlet started as a copy of `Set-AzureWebAppStorageContentFromStorage`, so I intended to have easy interchangeability between them, but this diverged more since then.